tcf_website/views/search.py

```diff
@@ -159,13 +166,19 @@ def format_instructors(results):
     return formatted
 
 
-def set_arguments(query, courses, instructors):
+def set_arguments(query, courses, instructors, courses_first):
     """Sets the search template arguments."""
     args = {
-        "query": query
+        "query": query,
+        "courses_first": courses_first
     }
     if not courses["error"]:
         args["courses"] = courses["results"]
     if not instructors["error"]:
         args["instructors"] = instructors["results"]
     return args
+
+
+def decide_order(courses, instructors):
+    """Determines whether courses or instructors should be displayed first in search page."""
+    return courses["highest_score"] > instructors["highest_score"]
```

Comment:
## Problem
I don't think comparing the highest scores is a safe way to decide ordering. Remember that each dataset is ranked by a DIFFERENT ranking algorithm, which means the scores are relative only to that specific algorithm and dataset. You need to normalize each set of scores before you compare them. 

## Solution
A simple solution would be to compute the z-score of the top result of each list. However, this makes the assumption that our course and instructor data is normally distributed (which is a fairly large assumption to make), but probably okay to make. You might want to do some research on normalization. I'm sure there better solutions out there. But, my intuition tells me that using z-score would be simple and effective.

## Pseudo-code for a z-score driven `decide_order`
```
def format_response(response):
      ...
      x_0 = results[0].get("_meta").get("score")
      μ = compute_mean_score(results)
      σ = compute_score_standard_deviation(results)
      formatted["z_0"] = (x_0 - μ) / σ
```
```
def decide_order(courses, instructors):
      return courses["z_0"] > instructors["z_0"]
```